out_writeapi.go

```diff
@@ -148,7 +151,7 @@ func parseMap(mapInterface map[interface{}]interface{}) map[string]interface{} {
 // it takes in the relevant queue of responses as well as boolean that indicates whether we should block the AppendRows function
 // and wait for the next response from WriteAPI
 // This function returns an error which is nil if the reponses were checked successfully and populated any were unsuccesful
```

Comment:
Please also document the new int return value.

Reply:
Done.